packages/artifact/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@actions/artifact",
+  "version": "0.1.0",
+  "preview": true,
+  "description": "Actions artifact lib",
+  "keywords": [
+    "github",
+    "actions",
+    "artifact"
+  ],
+  "homepage": "https://github.com/actions/toolkit/tree/master/packages/artifact",
+  "license": "MIT",
+  "main": "lib/artifact.js",
+  "types": "lib/artifact.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/actions/toolkit.git",
+    "directory": "packages/artifact"
+  },
+  "scripts": {
+    "audit-moderate": "npm install && npm audit --audit-level=moderate",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
I think tests should be part of this PR, at least covering the Upload APIs

Reply:
After way too many problems & issues I've added unit tests for everything. The structure matches the rest of this repo so tests are meant to be run from the root directory